tests/TestHelpers/GraphHelper.php

```diff
@@ -97,6 +97,17 @@ public static function getFullUrl(string $baseUrl, string $path): string {
 		return $fullUrl;
 	}
 
+	/**
+	 * @param string $baseUrl
+	 * @param string $path
+	 *
+	 * @return string
+	 */
+	public static function getBetaFullUrl(string $baseUrl, string $path): string {
+		$baseUrl = rtrim($baseUrl, "/");
+		return $baseUrl . '/graph/v1beta1/' . $path;
+	}
```

Comment:
what will we do when the path changes? change every occurrence of `getBetaFullUrl` to `getFullUrl`?

Reply:
I think, yes.